lab1/print_style.css

```diff
@@ -0,0 +1,83 @@
+body {
+  padding: 50px 100px;
+  background-color: white;
+  font-family: "Comic Sans MS", arial;
+  font-size: 100%;
+  text-align: justify;
+  text-justify: inter-word;
+}
+nav {
+  display:none;
+}
+
+a {
+  text-align: center;
+}
+
+a:link {
+    color: #000000;
```

Comment:
css has some colors you can reference by name, for example `black`
And what is `:link` for?
Also you can wrap things who belong together together by adding a comma. So in this case:
`a:link, a:visited`